salt/states/pkg.py

```diff
@@ -1053,8 +1053,16 @@ def installed(
     if not isinstance(version, six.string_types) and version is not None:
         version = str(version)
 
+    refreshed = False
```

Comment:
Sure, that'd be fine. That makes it sufficiently different from "refresh", I think.